docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,841 @@
+
+1. Your endpoint must be prepared to ACK a `PING` message
+2. Your endpoint must be set up to properly handle signature headers--more on that in [Security and Authorization](#DOCS_INTERACTIONS_SLASH_COMMANDS/security-and-authorization)
+
+If either of these are not complete, we will not validate your URL and it will fail to save.
+
+When you attempt to save a URL, we will send a `POST` request to that URL with a `PING` payload. The `PING` payload has a `type: 1`. So, to properly ACK the payload, return a `200` reponse with a payload of `type: 1`:
+
+```py
+@app.route('/', methods=['POST'])
+def my_command():
+    if request.json["type"] == 1:        
+        return jsonify({
+            "type": 1
+        })
+```
+
+You'll also need to properly set up [Security and Authorization](#DOCS_INTERACTIONS_SLASH_COMMANDS/security-and-authorization) on your endpoint for the URL to be accepted. Once both of those are complete and your URL has been saved, you can start receiving Interactions via webhook! At this point, your app will **no longer receive Interactions over the gateway**. If you want to receive them over the gateway again, simply delete your URL.
+
+An Interaction includes the `data` that the user sent in the command, as well as some metadata. It looks like this:
+
+```js
+{
+    "type": 2,
+    "token": "A_UNIQUE_TOKEN",
+    "member": { 
+        "user": {
+            "id": 53908232506183680,
+            "username": "Mason",
+            "avatar": "a_d5efa99b3eeaa7dd43acca82f5692432",
+            "discriminator": "1337",
+            "public_flags": 131141
+        },
+        "roles": ["539082325061836999"],
+        "premium_since": null,
+        "permissions": "2147483647",
+        "pending": false,
+        "nick": null,
+        "mute": false,
+        "joined_at": "2017-03-13T19:19:14.040000+00:00",
+        "is_pending": false,
+        "deaf": false 
+    },
+    "id": "786008729715212338",
+    "guild_id": "290926798626357999",
+    "data": { 
+        "options": [{
+            "name": "cardname",
+            "value": "The Gitrog Monster"
+        }],
+        "name": "cardsearch",
+        "id": "771825006014889984" 
+    },
+    "channel_id": "645027906669510667" 
+}
+```
+
+An explanation of all the fields can be found in our [data models](#DOCS_INTERACTIONS_SLASH_COMMANDS/data-models-and-types).
+
+Now that you've gotten the data from the user, it's time to respond to them.
+
+## Responding to an Interaction
+
+Interactions--both receiving and responding--are webhooks under the hood. So responding to an Interaction is just like sending a webhook request!
+
+When responding to an interaction received **via webhook**, your server can simply respond to the received `POST` request. You'll want to respond with a `200` status code (if everything went well), as well as specifying a `type` and `data`, which is an object with parameters the same as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook):
```

Comment:
As @advaith1 has pointed out for another section, this should probably point to the Interaction Response data type at the bottom, which is mentions "Not all message fields are currently supported."